packages/python/plotly/plotly/io/_kaleido.py

```diff
@@ -1,10 +1,17 @@
 from __future__ import absolute_import
 from six import string_types
 import os
+import sys
 import json
 import plotly
 from plotly.io._utils import validate_coerce_fig_to_dict
 
+if sys.version_info >= (3, 4):
+    from pathlib import Path, PurePath
+else:
```

Comment:
Let's remove this, as we're dropping support for 2.7 and 3.5 in the next Plotly.py release.